website/docs/architecture/chaos-control-plane.md

```diff
@@ -0,0 +1,21 @@
+---
+id: chaos-control-plane
+title: Chaos Control Plane
+sidebar_label: Chaos Control Plane
+---
+
+---
+
+<img src={require("../assets/control-plane.png").default} alt="Chaos Control Plane" />
+
+Chaos Control Plane consists of micro-services responsible for the functioning of the Chaos Center, the website-based portal that can be used for interacting with Litmus, apart from the CLI. Chaos Plane facilitates the creation and scheduling of chaos workflows, system observability during the event of chaos, and post-processing and analysis of experiment results. To achieve this, it uses four separate components:
+
+1. **Authentication Server:** A Golang microservice that is responsible for authorizing as well as authenticating the requests received from Chaos Center. It primarily serves the cause of user creation, user login, reset the password, and update user information.
+
+2. **Backend Server:** A GraphQL based Golang micro-service that serves the requests received from Chaos Center, by either querying the database for the relevant information or by fetching information from the Execution Plane.
+
+3. **Database:** A NoSQL MongoDB database micro-service that is accountable for storing users' information, past workflows, saved workflow templates, user projects, ChaosHubs, and GitOps details, among the other information.
+
+4. **Chaos Center:** A React.js based frontend application micro-service that serves as the user front-facing component that allows creation and scheduling of chaos workflows, system observability during the chaos injection, and chaos result analysis.  
```

Comment:
All together the interface is called Chaos Center. Can be accessed via UI, LitmusCTL or API.

Reply:
We can name this 'Web UI'